File: app/src/main/java/com/example/sanger/CartActivity.java
package com.example.sanger;

import androidx.appcompat.app.AppCompatActivity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageButton;

public class CartActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_cart);

        ImageButton btnBack = findViewById(R.id.btnBack);
        Button btnMasukanPesanan = findViewById(R.id.btnMasukanPesanan);

        btnBack.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                // TODO: Add back navigation logic here
                finish();
            }
        });

        btnMasukanPesanan.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                // TODO: Add navigation to payment method
                // Example: Intent intent = new Intent(CartActivity.this, PaymentMethodActivity.class);
                // startActivity(intent);
                android.widget.Toast.makeText(CartActivity.this, "Navigate to Payment Method - Add navigation logic", android.widget.Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: app/src/main/java/com/example/sangerfinal/CartActivity.kt
package com.example.sangerfinal

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import android.widget.ImageButton
import androidx.appcompat.app.AppCompatActivity

class CartActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_cart)

        // Set up back button
        val btnBack = findViewById<ImageButton>(R.id.btnBack)
        btnBack.setOnClickListener {
            finish() // This will close the current activity and return to ProfilActivity
        }

        // Set up Masukan Pesanan button
        val btnMasukanPesanan = findViewById<Button>(R.id.btnMasukanPesanan)
        btnMasukanPesanan.setOnClickListener {
            val intent = Intent(this, PaymentMethodActivity::class.java)
            startActivity(intent)
        }
    }
} 